frontend/src/models/associations/election.ts

```diff
@@ -0,0 +1,23 @@
+import { User } from "../user";
+
+export interface Choice {
+    id: string;
+    election: Election;
+}
+
+export interface Ballot {
+    id: string;
+    election: Election;
+    choices: Choice;
+}
+
+export interface Election {
+    id: string;
+    association: string;
+    name: string;
+    choices: Choice[];
+    registered_voters: User[];
```

Comment:
Snake or Camel case?

Snake will make it easier to work with the JSON (as the JSON fields are snake case). Nevertheless, I think we should follow Camel to suit the TS conventions.

In any case, we have to be consistent; it's `Association.myRole` a bit above

Reply:
Camel